core/src/test/kotlin/org/stellar/anchor/util/LogTest.kt

```diff
@@ -66,6 +77,55 @@ internal class LogTest {
     verify(exactly = 2) { logger.info(ofType(String::class)) }
   }
 
+  @Test
+  fun testInfoBPII() {
+    val slot = slot<String>()
+    every { logger.info(capture(slot)) } answers {}
+
+    val testBeanPII = TestBeanPII()
+    Log.infoB("Hello", testBeanPII)
+    verify(exactly = 2) { logger.info(ofType(String::class)) }
+    assertFalse(slot.captured.contains("fieldPII"))
+  }
```

Comment:
I've noticed the `slot` is not being used here... according with [this link](https://github.com/mockk/mockk/issues/352#issuecomment-652409698), you could use a list instead of a slot. It would look like this:

```suggestion
  @Test
  fun testInfoBPII() {
    val slotIMessages = mutableListOf<String>()
    every { logger.info(capture(slotIMessages)) } answers {}

    val testBeanPII = TestBeanPII()
    Log.infoB("Hello", testBeanPII)

    verify(exactly = 2) { logger.info(ofType(String::class)) }
    assertEquals("Hello", slotIMessages[0])
    val wantBean = """{
    "fieldNoPII": "no secret"
    }""".trimMargin()
    assertEquals(wantBean, slotIMessages[1])
  }
```